docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+cronitor:
+  build: .
+  container_name: cronitor
+  ports:
+    - 8434:8434
+  volumes:
+    - .:/opt/cronitor
+    - ./cronitor-log:/var/log/cronitor
```

Comment:
This line mount an directory from dockerhost into dockercontainer for persistent logs. you can delete container and start again a new, logfiles where keept